app/views/lists/index.html.erb

```diff
@@ -19,6 +22,13 @@
         <td><%= link_to 'Show', list %></td>
         <td><%= link_to 'Edit', edit_list_path(list) %></td>
         <td><%= link_to 'Destroy', list, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <% if current_user%>
+          <% if current_user.homepage.lists.include? list %>
```

Comment:
We should probably try to avoid this kind of thing where possible - the `current_user` logic in the views, I mean.

Also pretty sure we can figure out what lists are on the homepage in the controller.